common/on_actions/01_pop_on_actions.txt

```diff
@@ -0,0 +1,13 @@
+#When adding effects directly in on_actions always consider if it would be best to do so through a scripted effect. on_actions with a lot of script in them are harder to get an overview of.
```

Comment:
We should pu it in 00_pop_on_actions, the on_yearly_pulse is already defined there

Reply:
on_actions are additive